pyproject.toml

```diff
@@ -40,15 +44,13 @@ dynamic = [
 name = "epitools"
 optional-dependencies = {dev = [
     "black",
-    "flake8",
-    "isort",
-    "napari",
+    "build",
+    "mypy",
     "pre-commit",
     "pyqt5",
-    "pytest",
-    "pytest-cov",
-    "pytest-qt",
+    "ruff",
     "tox",
+    "twine",
```

Comment:
I think we need the `pytest` dependency still? And `build` and `twine` could be moved to a separate `deploy` section

Reply:
Will remove `build` and `twine`. As for `pytest` we then have to add `pytest-qt` etc etc, so will leave.